src/render/DmnPropertiesPanel.js

```diff
@@ -22,138 +10,25 @@ import { PanelPlaceholderProvider } from './PanelPlaceholderProvider';
    * @param {djs.model.Base|Array<djs.model.Base>} [props.element]
+   * @param {Array} props.groups
    * @param {Injector} props.injector
-   * @param { (djs.model.Base) => Array<PropertiesProvider> } props.getProviders
    * @param {Object} props.layoutConfig
    * @param {Object} props.descriptionConfig
    * @param {Object} props.tooltipConfig
    */
 export default function DmnPropertiesPanel(props) {
   const {
     element,
+    groups,
     injector,
-    getProviders,
     layoutConfig,
     descriptionConfig,
     tooltipConfig
   } = props;
 
-  const canvas = injector.get('canvas');
-  const elementRegistry = injector.get('elementRegistry');
   const eventBus = injector.get('eventBus');
 
-  const [ state, setState ] = useState({
-    selectedElement: element
-  });
-
-  const selectedElement = state.selectedElement;
-
-  /**
-   * @param {djs.model.Base | Array<djs.model.Base>} element
-   */
-  const _update = (element) => {
-
-    if (!element) {
-      return;
-    }
-
-    let newSelectedElement = element;
-
-    // handle labels
-    if (newSelectedElement && newSelectedElement.type === 'label') {
-      newSelectedElement = newSelectedElement.labelTarget;
-    }
-
-    setState({
-      ...state,
-      selectedElement: newSelectedElement
-    });
-
-    // notify interested parties on property panel updates
-    eventBus.fire('propertiesPanel.updated', {
-      element: newSelectedElement
-    });
-  };
-
-  // (2) react on element changes
-
-  // (2a) selection changed
-  useEffect(() => {
-    const onSelectionChanged = (e) => {
-      const { newSelection = [] } = e;
-
-      if (newSelection.length > 1) {
-        return _update(newSelection);
-      }
-
-      const newElement = newSelection[0];
-
-      const rootElement = canvas.getRootElement();
-
-      if (isImplicitRoot(rootElement)) {
-        return;
-      }
-
-      _update(newElement || rootElement);
-    };
-
-    eventBus.on('selection.changed', onSelectionChanged);
-
-    return () => {
-      eventBus.off('selection.changed', onSelectionChanged);
-    };
-  }, []);
-
-  // (2b) selected element changed
-  useEffect(() => {
-    const onElementsChanged = (e) => {
-      const elements = e.elements;
-
-      const updatedElement = findElement(elements, selectedElement);
-
-      if (updatedElement && elementExists(updatedElement, elementRegistry)) {
-        _update(updatedElement);
-      }
-    };
-
-    eventBus.on('elements.changed', onElementsChanged);
-
-    return () => {
-      eventBus.off('elements.changed', onElementsChanged);
-    };
-  }, [ selectedElement ]);
-
-  // (2c) root element changed
-  useEffect(() => {
-    const onRootAdded = (e) => {
-      const element = e.element;
-
-      if (isImplicitRoot(element)) {
-        return;
-      }
-
-      _update(element);
-    };
-
-    eventBus.on('root.added', onRootAdded);
-
-    return () => {
-      eventBus.off('root.added', onRootAdded);
-    };
-  }, [ selectedElement ]);
```

Comment:
This logic can be safely removed as we refuse to render implicit root in _render